cmd/juju/commands/bootstrap.go

```diff
@@ -1025,6 +1025,11 @@ func (c *bootstrapCommand) bootstrapConfigs(
 	if err != nil {
 		return bootstrapConfigs{}, errors.Annotate(err, "constructing bootstrap config")
 	}
+
+	if len(cloud.CACertificates) > 0 {
```

Comment:
If the cloud has certificates then the controller is going to need access to those certificates, ok, I understand this.